Releases failing on GitHub Actions

#### Summary

Even though the main CI is passing, the release workflow on GitHub Actions is failing:
https://github.com/googlemaps/android-maps-utils/actions/runs/1012280565

>release
Error: Command failed with exit code 1: ./gradlew build --info --stacktrace
>
>FAILURE: Build failed with an exception.
>
>* What went wrong:
A problem occurred configuring root project 'android-maps-utils'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find org.jetbrains.kotlin:kotlin-gradle-plugin:2.2.5.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-gradle-plugin/2.2.5/kotlin-gradle-plugin-2.2.5.pom
       - https://repo.maven.apache.org/maven2/org/jetbrains/kotlin/kotlin-gradle-plugin/2.2.5/kotlin-gradle-plugin-2.2.5.pom
       - https://plugins.gradle.org/m2/org/jetbrains/kotlin/kotlin-gradle-plugin/2.2.5/kotlin-gradle-plugin-2.2.5.pom
     Required by:
         project :
>
>* Try:
>Run with --debug option to get more log output. Run with --scan to get full insights.

What's confusing is that it should be looking for:

```
classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20"
```

I've re-run it twice and it keeps failing with the same error.

@arriolac Any idea why this is happening?

#### Expected behavior

Release should build and publish

#### Observed behavior

Release fails with above error

#### Environment details

GitHub CI

#### Steps to reproduce

See above GitHub CI run

--- a/.releaserc
+++ b/.releaserc
@@ -7,7 +7,7 @@ plugins:
     - replacements:
         - files:
             - "./build.gradle"
-          from: "version = '.*'"
+          from: "\bversion = '.*'"
           to: "version = '${nextRelease.version}'"
 
         - files:

